File: lua/lualine/themes/arctic.lua
local colors = {
	normal  = '#5ca1c1',
	insert  = '#8fbc95',
	visual  = '#7c76c0',
	replace = '#bf616a',
	command = '#d08770',

	fg      = '#e5e9f0',
	bg      = '#222834',
	bg_alt  = '#0e1015',
}

return {
	normal = {
		a = { fg = colors.bg, bg = colors.normal },
		b = { fg = colors.normal, bg = colors.bg },
		c = { fg = colors.fg, bg = colors.bg_alt },
	},
	insert = { a = { fg = colors.bg, bg = colors.insert } },
	visual = { a = { fg = colors.bg, bg = colors.visual } },
	command = { a = { fg = colors.bg, bg = colors.command } },
	replace = { a = { fg = colors.bg, bg = colors.replace } },

	inactive = {
		a = { bg = colors.bg, fg = colors.normal },
		b = { bg = colors.bg, fg = colors.bg_alt },
		c = { bg = colors.bg, fg = colors.bg_alt },
	},
}

File: schemes/arctic/arctic.go
package arctic

import "github.com/jimmykodes/colorschemes"

var (
	fg        = colorschemes.ColorMustFromHex("#E5E9F0")
	bg        = colorschemes.ColorMustFromHex("#222834")
	red       = colorschemes.ColorMustFromHex("#BF616A")
	orange    = colorschemes.ColorMustFromHex("#D08770")
	yellow    = colorschemes.ColorMustFromHex("#EBCB8B")
	green     = colorschemes.ColorMustFromHex("#8fbc95")
	cyan      = colorschemes.ColorMustFromHex("#88e0d0")
	blue      = colorschemes.ColorMustFromHex("#5ca1c1")
	purple    = colorschemes.ColorMustFromHex("#7c76c0")
	magenta   = colorschemes.ColorMustFromHex("#B48EAD")
	white     = colorschemes.ColorMustFromHex("#ceeff4")
	gray      = colorschemes.ColorMustFromHex("#6b7486")
	dark_gray = colorschemes.ColorMustFromHex("#3B4252")
	black     = bg.Darken(0.1)
)

var Arctic = &colorschemes.Colorscheme{
	Metadata: colorschemes.Metadata{
		Name:  "arctic",
		Theme: colorschemes.Dark,
	},

	Fg:             fg,
	Bg:             bg,
	BgDark:         bg.Darken(.1),
	BgLight:        bg.Lighten(.1),
	HighlightAltBg: bg.Lighten(.2),

	Red:      red,
	Orange:   orange,
	Yellow:   yellow,
	Green:    green,
	Cyan:     cyan,
	Blue:     blue,
	Purple:   purple,
	Magenta:  magenta,
	White:    white,
	Gray:     gray,
	DarkGray: dark_gray,

	LightBlack:  dark_gray,
	LightRed:    red.Lighten(.20),
	LightGreen:  green.Lighten(.20),
	LightYellow: yellow.Lighten(.20),
	LightBlue:   blue.Lighten(.20),
	LightPurple: purple.Lighten(.20),
	LightCyan:   cyan.Lighten(.20),
	LightWhite:  white.Lighten(.20),
	LightGray:   gray.Lighten(.20),

	Hint:    magenta,
	Info:    blue,
	Warning: yellow,
	Error:   red,
	Accent:  dark_gray,
	Search:  magenta,
	Replace: orange,
	Comment: gray,
	Context: gray,

	CursorFg:   bg,
	CursorBg:   fg,
	CursorLine: bg.Darken(.05),

	SignDiff:         yellow,
	SignDiffBg:       yellow.Darken(.25),
	SignAdd:          green,
	SignAddBg:        green.Darken(.25),
	SignStagedAdd:    green.Lerp(bg, .25),
	SignChange:       blue,
	SignChangeBg:     blue.Darken(.25),
	SignStagedChange: blue.Lerp(bg, .25),
	SignDelete:       red,
	SignDeleteBg:     red.Darken(.25),
	SignStagedDelete: red.Lerp(bg, 0.25),
}
